lib/global_uid.rb

```diff
@@ -9,6 +9,8 @@ module GlobalUid
   class NoServersAvailableException < StandardError ; end
   class ConnectionTimeoutException < StandardError ; end
   class TimeoutException < StandardError ; end
+  class InvalidIncrementException < StandardError ; end
```

Comment:
Do we really need two exception classes? One might suffice.

Reply:
The intent was to highlight when data corruption has occurred(InvalidIncrementException) and when it is prevented (InvalidIncrementPreventionException), allowing clients to differentiate between the two and clean up data if it's encountered. 

Maybe I was over thinking it :)

Reply:
I am still unsure what the two exceptions mean. Does one mean something bad has happened, and the other means that something bad was about to happen but didn’t?